### PR TITLE
Merge master branch to nightly

### PR DESCRIPTION
**Pull request by mthalman**

### CODE REVIEW

File: README.aspnet.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 To provide a consistent FQDNs, the data endpoint will be changing from *.cdn.mscr.io to *.data.mcr.microsoft.com
 
 For more info, see [MCR Client Firewall Rules](https://aka.ms/mcr/firewallrules).
+
```
Comment (MichaelSimons) [inline README.aspnet.md:6]: I'm not understanding why this change is showing up?  It appears in nightly already.

Comment (mthalman) [reply README.aspnet.md:6]: Me neither.
